### ISSUE TITLE
Initial page

### ISSUE THREAD

**Issue by rxaviers** [jquery/api.globalizejs.com#1]
## Goal

To create `entries/*xml`.
## General idea for content
- https://github.com/jquery/globalize/#api
- https://github.com/jquery/globalize/tree/master/doc/api
## Other stuff
- [ ] List api.globalizejs.com in http://api.jquery.com/
## Reference for implementation
- https://github.com/jquery/api.jquery.com
- https://github.com/jquery/api.jqueryui.com
- https://github.com/jquery/api.jquerymobile.com
- https://github.com/jquery/api.qunitjs.com

**Comment by arthurvr** [comment]
Sounds really good :+1:.

Globalize already has really nice documentation living in `doc/api` in the repo, I guess porting those over to xml would be an awesome start. Later on, is the goal that the files are going to get removed from the main repo?

**Comment by scottgonzalez** [comment]
> Later on, is the goal that the files are going to get removed from the main repo?

Yes. Otherwise there would be duplicated content and the likelihood of them being out of sync is extremely high.

**Comment by arthurvr** [comment]
> Otherwise there would be duplicated content and the likelihood of them being out of sync is extremely high.

Yeah, thought so. :+1:

## 

What's the current status of this in jquery-wp-content? I know we started working on globalizejs.com at some point, but I don't think we started working on this one right?

**edit: nevermind, just found the jquery-wp-content branch.**
